### DISCUSSION TITLE
getInitialProps don't work when page reload and directly go site link

### DISCUSSION THREAD

**Discussion by sahinme** [vercel/next.js#15801] [Help]
i am using getInitialProps in pages for initial data populating. So anyone go website directly or press f5 getInitialProps don't work. Only working when next/link or next/router navigate.
How can i fix this ?

`PostDetail.getInitialProps = async (ctx) => {
  await ctx.store.execSagaTasks(ctx.isServer, (dispatch) => {
    const { post } = ctx.query;
    dispatch(
      getPostDetailRequest({
        slug: post,
        loaderStart: true,
        token: parseCookies(ctx).token,
        isServer: true
      })
    );
  });
  return {};
};`

**Comment by jamesmosier** [comment]
Could you share any more code that helps to demonstrate the issue?

**Reply by sahinme** [reply]
Actually i fixed that. i removed this line from get request method :
`process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";`
